styles/styles.css

```diff
@@ -6,3 +6,148 @@
  * 
  *
  */
+
+/* main > div {
+  width: 100%!important;
+  margin: 0 auto;
+  max-width: 1260px;
+  padding-left: 70px!important;
+  padding-right: 70px!important;
+} */
+
+body {
+  line-height: 21px;
+}
+
+body table {
+  font-size: 16px;
+  border-spacing: 0;
+}
+
+table, th, td {
+  padding: 9px 10px;
+  border: 1px solid #bdbdbd!important;
+  border-collapse: collapse!important;
+}
+
+table th {
+  background-color: #ededed;
+}
+
+.columns {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.accordion dd div {
+  padding-top: 10px;
+}
+
+.content {
+  display: block;
+  max-width: var(--grid-container-width);
+  margin: 0 auto;
+}
+
+/*
+* title
+*/
+
+.page-title {
```

Comment:
My understanding is that only global styles should be present here. 
Could you please check whether styles related to `page-title` should go here.

Reply:
For now the title is implemented via the `metadata-section` Milo block. The existing parameters of the existing block does not allow to replicate the exact style of the title from the existing helpx internal pages.
So the need to add specific styles for this block and also the need to use `!important` for some parameters.